### PR TITLE
doc, os/bluestore: Zoned storage related cleanup.

### PR DESCRIPTION
**Pull request by agayev**
Update zoned-storage.rst, prefix all zone-related functions with zoned, and move them to one place.

Signed-off-by: Abutalib Aghayev agayev@psu.edu

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Comment by agayev** [comment]
@tchaikov This is a miscellaneous cleanup; no new functionality was added.  (Do you think I should rename all zoned_ stuff to zbd_?)

File: doc/dev/zoned-storage.rst [hunk 1]
```diff
@@ -34,21 +34,21 @@ Currently we can perform basic RADOS benchmarks on an OSD running on an HM-SMR
 drives, restart the OSD, and read the written data, and write new data, as can
 be seen below.
 
-Please contact Abutalib Aghayev <agayev@cs.cmu.edu> for questions.
+Please contact Abutalib Aghayev <agayev@psu.edu> for questions.
 
 ::
    
-  $ zbc_info /dev/sdc
+  $ sudo zbd report -i -n /dev/sdc
```
Comment (tchaikov) [inline doc/dev/zoned-storage.rst:41]: nit, might want to drop the trailing space?

**Review by tchaikov — APPROVED** [review]

**Comment by tchaikov** [comment]
> @tchaikov This is a miscellaneous cleanup; no new functionality was added. (Do you think I should rename all zoned_ stuff to zbd_?)

zbd is short for zoned block device, while in this context, we are always interacting with block device. and "zoned" alone is a well-known name. for instance, https://www.kernel.org/doc/html/latest/admin-guide/device-mapper/dm-zoned.html . so unless you think "zoned" could be misleading or confusing, i think it's good enough.

File: doc/dev/zoned-storage.rst (hunk 1 above)
Comment (agayev) [reply doc/dev/zoned-storage.rst]: Done.

**Comment by agayev** [comment]
> > @tchaikov This is a miscellaneous cleanup; no new functionality was added. (Do you think I should rename all zoned_ stuff to zbd_?)
> 
> zbd is short for zoned block device, while in this context, we are always interacting with block device. and "zoned" alone is a well-known name. for instance, https://www.kernel.org/doc/html/latest/admin-guide/device-mapper/dm-zoned.html . so unless you think "zoned" could be misleading or confusing, i think it's good enough.

If you think it is good enough, then I'll keep it as is.  Thanks.

**Comment by tchaikov** [comment]
https://shaman.ceph.com/builds/ceph/wip-kefu-testing-2020-11-05-1414/e6820d4151f7302ab23305157765dd0b4fc4e835/

**Comment by tchaikov** [comment]
Signed-off-by: Kefu Chai <kchai@redhat.com>

**Comment by tchaikov** [comment]
jenkins test make check

**Pull request merged**